### PR TITLE
Add Weekly report for 16th May 2022

### PR DESCRIPTION
**Pull request by philberty**

### CODE REVIEW

**Comment by dkm** [comment]
Links to rendered docs, for the lazy like me :

https://github.com/Rust-GCC/Reporting/blob/phil/weekly-16-05-2022/2022-05-13-community-call.md
https://github.com/Rust-GCC/Reporting/blob/phil/weekly-16-05-2022/2022-05-16-report.org

**Review by CohenArthur — APPROVED** [review]

File: 2022-05-16-report.org [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+const TEST: *mut u8 = 123 as *mut u8;
+
+fn test() {
+    let a = TEST;
+}
+#+END_SRC
+
+*** Privacy violations
+
+@arthur
```
Comment (philberty) [inline 2022-05-16-report.org:127]: @CohenArthur do you want to fill this out for us :D

File: 2022-05-16-report.org [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+- Add more linting and restrictions to our privacy pass
+- Continue bugfixing in aim of compiling our goal test case
+- Continue work on metadata exports
+
+** Detailed changelog
+
+*** Match Expression
+
+ @dfaust
```
Comment (philberty) [inline 2022-05-16-report.org:100]: @dafaust do you want to fill out a section on the match expression updates for us?

File: 2022-05-16-report.org (hunk 1 above)
Comment (CohenArthur) [reply 2022-05-16-report.org:127]: Yes, will do! Thought this PR was only about adding community call meeting notes :) I'll fill this out

**Pull request merged**